wix-bookings-v2/wix-bookings-v2/SessionsView/guides/1. Introduction.md

```diff
@@ -10,3 +10,8 @@ The sessions view consists with only single sessions or expanded, recurring sess
 
 The sessions view has an end date, which is set to be at least a year into the future.  
 Periodically, the view is automatically extended, to preserve its future, 1 year, duration.
+
+## Limitations
```

Comment:
How about explaining what the difference between Session Feed and Session View services? When do you use which? How are they related?  

And I would put this paragraph also in the Intro for Session Feed when you write it.